tests/Doctrine/Tests/DBAL/Driver/PDOSqlsrv/DriverTest.php

```diff
@@ -16,4 +19,22 @@ protected function createDriver()
     {
         return new Driver();
     }
+
+    protected function checkForSkippingTest(AbstractSQLServerDriver $driver) : void
+    {
+        if (extension_loaded('pdo_sqlsrv') && $driver instanceof Driver) {
```

Comment:
As long as this test uses a real connection, it should be part of the functional suite, where the availability of the extension will be checked in `setUp()`.